01/implement-most-frequent-k-elements.instructor.md

```diff
@@ -0,0 +1,26 @@
+# Instructor: Implement Most Frequent K Elements
+
+An example of a working implementation:
+
+```python
+#Time Complexity: O(N log(N)) where N is the size of the array
+# Space Complexity: O(K) where K is the number of unique integers in a given array
```

Comment:
If picking a different variable, I'd go with M rather than K, to avoid confusion with the `k` param. But I'd probably stick with N for everything.

In the worst case, the number of unique numbers in the array is the same as the length of the array itself, so everything can be expressed in terms of the length of the array.

Reply:
✅